### ISSUE TITLE
Improve app.js to implement logic for the application.

### ISSUE THREAD

**Issue by theinit01** [gdsc-jssstu/CipherBox#13]
The app.js file in the /js directory, is supposed to have a encryptString and a decryptString function each. This file imports logic for algorithm implementations from the files in /crypto directory.

**Comment by devesh-mishra13** [comment]
I am interested in contributing in this issue but can you explain a little bit what changes are needed to be done in app.js

**Comment by theinit01** [comment]
## Contributing to CipherBox

Thank you for your interest in contributing to CipherBox! To help you get started, here's an overview of what can be done in the `app.js` file of our project.

### Tasks in `app.js`

1. **Integration of Cryptographic Algorithms:** The `app.js` file serves as the central point for integrating various cryptographic algorithms available in the `/crypto` directory. Your task is to make sure that the functions in `app.js` effectively utilize the encryption and decryption logic from the respective files in the `/crypto` directory. You can work on managing the flow of data, user input, and key handling.

2. **User Interface Enhancements:** If you have expertise in front-end development, you can contribute to improving the user interface. This includes refining the layout, enhancing user experience, and making the application more user-friendly while ensuring it works seamlessly with the cryptographic functions.

3. **Error Handling and Validation:** Help ensure that the application provides appropriate error handling and validation for user inputs, especially in the context of cryptographic operations. This includes handling invalid data, displaying informative error messages, and securing key management.

### Getting Started

If you're new to contributing to open-source projects, don't worry. We have a set of guidelines and resources in our repository to help you. Check our [Contributing Guidelines](./CONTRIBUTING.md) for detailed information on the contribution process.

Your contributions are highly valued, and together, we can make CipherBox even better! 🚀

**Comment by urz-ahmed** [comment]
hey there @theinit01 I can surely work in this as I have a knowledge of crytography as well. Could you assign me this issue under hacktoberfest-accepted, hacktoberfest tag

**Comment by theinit01** [comment]
Yeah sure, @urz-ahmed. The repository is tagged hacktoberfest so there's no need for labelling the issues under hacktoberfest tag. Your PR will be counted as valid if accepted. I am assigning this to you. Looking forward for you PR.

**Comment by urz-ahmed** [comment]
Okay sure. Thanks
